values.md

```diff
@@ -1,6 +1,6 @@
-### Profits are exhaust.
+### We’re here to build something that matters.
```

Comment:
```suggestion
### We’re here to build things that matter.
```

Ok, to start -- I freaking love this. Your statement about PJB really resonates, and the "reactionary" element really resonates, too ("_It’s too anti-$$ while not being clearly for something else._") This is a big improvement.

But, "something that matters" feels too singular and non-descript. We all build _things_ that matter, and we aspire to build more things that matter. And, IMO, that's part of the builders mindset -- we're not really ever done, there is just more to build.

My 2c.

Reply:
love this. an earlier version of my change proposal read more like this. i'm going to update this now.